src/_pytest/terminal.py

```diff
@@ -143,6 +143,12 @@ def pytest_addoption(parser):
         default="progress",
     )
 
+    parser.addini(
+        "progress_display_mode",
```

Comment:
Instead of a new option, how about adding a new `count` option to `console_output_style`?